docs/PacketFence_Aerohive_Quick_Install_Guide.asciidoc

```diff
@@ -148,61 +147,62 @@ Step 2: Configure Network and PacketFence
 Configuring Aerohive access point in PacketFence
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We will nowneed to create a new switch, in PacketFence to be able to manage the terminals behind the Aerohive APs.
+We will now need to create a new switch, in PacketFence to be able to manage the endpoints behind the Aerohive APs
 
 Logon to your PacketFence interface::
 * From _Configuration / Policies and Access Control / Switches /_
-* On the line where there is the _default_, on the right, Click on _CLONE_.
+* On the line where there is the _default_, on the right, Click on _CLONE_
 
 
 image::docs/images/pfaerohive09.png[scaledwidth="100%",alt="New switch configuration",align=right]
 
+
 '''
 
 
 In _Definition_ ::
-* _IP Address/MAC Address/Range (CIDR)_, give the network adress of your _Production_ network,for us, it will be *172.16.1.0/24*
-* _Description_, give a description so you can quickly see what it is about,
-* From the _Type_ list, select *AeroHIVE AP*.
-
-NOTE: *TIP*: You can type directly the name you are looking forin the field
-
+* _IP Address/MAC Address/Range (CIDR)_, give the network address of your _Production_ network; For us, it will be *172.16.1.0/24*
+* _Description_, give a description so you can quickly see what it is about
+* From the _Type_ list, select *Aerohive AP*
 * From _Mode_ select *Production*
-* _Switch Group_ by defaultset to *None*.
+* _Switch Group_ by default set to *None*
 * _Deauthentication Method_ set to *RADIUS*
 * Click *SAVE*
 
 image::docs/images/pfaerohive10.png[scaledwidth="100%",alt="New switch configuration",align=right]
 
+
 '''
 
 
 In _Role_ ::
-* Set all VLans ID in each roles.
+* Set all VLans ID in each roles
 
 image::docs/images/pfaerohive11.png[scaledwidth="100%",alt="New switch configuration",align=right]
 
+
 '''
 
 
 In _RADIUS_ ::
-* Modify the secret passphrase previously sets in the AeroHive Hive Manager,
+* Modify the secret passphrase previously sets in the Aerohive HiveManager
 * Click on *SAVE*
 
-This end the PacketFence configuration.
+This ends the PacketFence configuration
 
 '''
 
 Web Auth Configuration
 ----------------------
 
-In this part we will describe the WebAuth configuration for Aerohive APs.
+In this part we will describe the WebAuth configuration for Aerohive APs
 
 
 Assumption :
 ~~~~~~~~~~~~
-We will use the same previous contexte, but we will have only one VLAN, the Production (11)
-Keep in mind that we will also use the natvite VLAN tag (1) as default one.
+
+In this part, the default VlAN must be the native VLAN.
+This way, the AP and the others networks equipments will be able to manage VLANs 
```

Comment:
Using this configuration, the access point and other network equipments will be allowed to manage ...

are you sure about that? I mean, what are you trying to say by "will be able to manage VLANs". It's not clear to me so it won't be for readers.